packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -251,22 +251,18 @@ const useTimeOption = (raw: RawTimeRange, quickOptions: TimeOption[]): TimeOptio
   }, [raw, quickOptions]);
 };
 
-const getStyles = stylesFactory((theme: GrafanaTheme2, isReversed, hideQuickRanges, isContainerTall) => {
+const getStyles = stylesFactory((theme: GrafanaTheme2, isReversed, hideQuickRanges, isContainerTall, isFullscreen) => {
   return {
     container: css`
       background: ${theme.colors.background.primary};
       box-shadow: ${theme.shadows.z3};
       position: absolute;
       z-index: ${theme.zIndex.dropdown};
-      width: 546px;
+      width: ${isFullscreen ? '546px' : '262px'};
```

Comment:
I realize this is leaving out the `only screen` aspect of this, but I think this kind of styling would be universal to all media types.